dev/com.ibm.ws.jaxrs.2.1.sse_fat/test-applications/SseClientBehaviorApp/src/jaxrs21sse/clientbehavior/SseClientBehaviorTestServlet.java

```diff
@@ -291,7 +292,11 @@ public void run() {
 
             source.open();
             _log.info("client source open");
-            assertTrue("Completion listener runnable was not executed", executionLatch.await(30, TimeUnit.SECONDS));
+            if (receiveCompletion) {
+                assertTrue("Completion listener runnable was not executed", executionLatch.await(30, TimeUnit.SECONDS));
```

Comment:
It should be true for all of EE 8, 9 and 10.  This is changed to EE 11 behavior for all other EE levels.